<!DOCTYPE html>
<h2>Dynamics and distribution of MBD proteins on DNA</h2>
<br>
<div>
    <p>Members of the methyl-cytosine binding domain family of proteins contain a small ~60 amino acid domain that
        specifically recognizes symmetrically methylated CpG dinucleotides. These proteins either recruit co-regulatory
        complexes (i.e. NuRD) to methylated DNA or, in the case of MBD4, identify and repair TpG mismatches. Vertebrate
        genomes contain regions of relatively increased CpG content (CpG islands) often found associated with the
        promoter
        regions of genes. Unlike transcription factors which recognize isolated binding sites, MBD proteins are
        recruited to
        CpG islands with hundreds of potential binding sites. We have previously shown that subtle structural
        differences
        between MBD proteins alter rates of exchange between neighboring CpGs.<a class="link ref" target="_blank"
            href="http://nar.oxfordjournals.org/content/42/17/11218.long"><sup>ref</sup></a> We are collaborating with
        the
        <a class="link" target="_blank" href="https://www.physics.ncsu.edu/wang/">Hong Wang Lab</a> at NC State to
        analyze
        this behavior on “DNA
        tightropes” with single molecule fluorescence. Using a combination of Z-exchange NMR spectroscopy, single
        molecule
        fluorescence, FRET, and binding studies, we are addressing key questions about the function and distribution of
        MBD
        proteins:
    </p>
    <br>
    <ul>
        <li>Why do MBD2 and MBD3 selectively localize to methylated and unmethylated CpG islands?</li>
        <li>How does methylation influence localization and exchange kinetics of the MBD proteins?</li>
        <li>Do the intrinsically disordered regions of MBD2 and MBD3 influence behavior on DNA?</li>
    </ul>
    <br>
    <span class="left side"><a href="../assets/images/SlidingModelnew-300x175-1.png" target="_blank"><img
                src="../assets/images/SlidingModelnew-300x175-1.png" alt=""></a>
        <p class="image-2">Transcription factors identify binding sites through a combination of free diffusion and 1D
            sliding, hopping, and jumping mechanisms. MBD2 binds to densely methylated CpG islands and recruits the NuRD
            chromatin remodeling complex. We hypothesize that rapid exchange between nearby CpG sites is critical to the
            function of MBD proteins.</p>
    </span>
    <span class="left side"><a href="../assets/images/ezgif.com-gif-maker-2.gif" target="_blank"><img
                src="../assets/images/ezgif.com-gif-maker-2.gif" alt=""></a>
        <p class="image-2">Using the DNA tightrope assay developed by the Wang laboratory, we can visualize MBD2
            diffusing
            along DNA (courtesy of <a class="link" target="_blank"
                href="https://www.physics.ncsu.edu/people/pd_pan.php">Hai
                Pan</a>, NC State University).</p>
    </span>
</div>
<br>

<span class="side"><a href="../assets/images/MBD4_Zexchange.png" target="_blank"><img
            src="../assets/images/MBD4_Zexchange.png" alt=""></a>
    <p class="image-2 credit">Ninad M. Walavalkar et al. Nucl. Acids Res. 2014;42:11218-11232 (modified) © The Author(s)
        2014.
        Published by Oxford University Press on behalf of Nucleic Acids Research.</p>
    <div class="side no-border">
        <p class="image-2">(a) Select resonances in the 2D<sup>15</sup> show two distinct chemicals shifts when bound to
            a
            mixture of two unique mCpG sites. Cross peaks (AB and BA) between these resonances indicate chemical
            exchange
            between the two sites during the pulse sequence between t<sub>1</sub> and t<sub>2</sub>.</p><br>
        <p class="image-2">(b) Inserting an additional delay between t<sub>1</sub> and t<sub>2</sub> during the HSQC
            pulse
            sequence while magnetization is along the Z axis allows for the cross peaks to build in intensity
            (Z-exchange
            spectroscopy). The rate of exchange between binding sites can be determined by fitting the intensities of
            AA,
            AB, BA, and AB cross peaks.</p>
    </div>
</span>