<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="stylesheet" href="../styles/styles.css">
    <link rel="stylesheet" href="../styles/structures.css">
    <link rel="stylesheet"
        href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
    <script src="../scripts/navbar.js" defer></script>
    <script src="../scripts/structures.js" defer></script>
    <script src="https://code.jquery.com/jquery-3.6.3.min.js"></script>
    <script src="../jsmol/JSmol.min.nojq.js"></script>
    <link rel="apple-touch-icon" sizes="180x180" href="../icons/apple-touch-icon.png">
    <link rel="icon" type="image/png" sizes="32x32" href="../icons/favicon-32x32.png">
    <link rel="icon" type="image/png" sizes="16x16" href="../icons/favicon-16x16.png">
    <link rel="manifest" href="../icons/site.webmanifest">
    <link rel="mask-icon" href="../icons/safari-pinned-tab.svg" color="#5bbad5">
    <meta name="msapplication-TileColor" content="#da532c">
    <meta name="theme-color" content="#ffffff">
    <title>Williams Lab | Structures</title>
</head>

<body>
    <nav class="nav">
        <img src="/assets/images/1Dspectrum_black.png" alt="" class="logo">
        <span class="logo-name">Williams Lab</span>
        <span class="material-symbols-outlined menubtn">menu</span>
        <ul class="navlist">
            <li><a href="/">Home</a></li>
            <li><a href="/pages/research.html">Research</a></li>
            <li><a href="/pages/structures.html" class="current">Structures</a></li>
            <li><a href="/pages/publications.html">Publications</a></li>
            <li><a href="/pages/people.html">People</a></li>
        </ul>
    </nav>
    <div class="openmenu close">
        <ul>
            <li><a href="/">Home</a></li>
            <li><a href="/pages/research.html">Research</a></li>
            <li><a href="/pages/structures.html" class="current">Structures</a></li>
            <li><a href="/pages/publications.html">Publications</a></li>
            <li><a href="/pages/people.html">People</a></li>
        </ul>
    </div>
    <div class="page-container">
        <div id="title" class="content-container">
            <h1>Structures</h1>
        </div>
        <hr>
        <div class="buttons">
            <span class="material-symbols-outlined prev">chevron_left</span>
            <span class="pageNum"></span>
            <span class="material-symbols-outlined next">chevron_right</span>
        </div>
        <div id="cMBD2" class="content-container">
            <span class="right"><img src="../assets/images/MBD2intfc.png" alt="">
                <p class="image-2">Two key arginine residues form bidentate hydrogen bonds with the
                    symmetrically related
                    guanosine bases of
                    the CpG dinucleotide. A conserved aspartate residue stabilizes one arginine, while a tyrosine
                    interacts
                    with the methyl-cytosine.</p>
            </span>
            <span class="right">
                <script class="jmol" src="../structures/2ky8.js"></script>
                <p class="image-1">The MBD2<sub>MBD</sub> binds in the major groove of DNA. Two
                    β-strands form a long
                    finger-like projection
                    that
                    extends down the major groove and provides most of the contacts with DNA.</p>
            </span>
            <h2>cMBD2:mCpG</h2>
            <p>We solved the structure of the chicken MBD2<sub>MBD</sub> bound to a methylated DNA sequence from the
                chicken ρ globin promoter (<a class="link"
                    href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2KY8">2KY8</a>). The domain consists of
                a three-stranded β sheet which extends down the major groove to make base-specific contacts with DNA. A
                small C-terminal α-helix and N- and C- terminal loop regions pack against the opposite face of this β
                sheet. Two highly conserved arginine residues form bidentate hydrogen bonds with the symmetrically
                related guanosine bases of the mCpG dinucleotide. The aliphatic portion of the arginine side chains
                packs against the methyl groups of the preceding 5mC. A critical tyrosine residue points towards one 5mC
                such that the hydroxyl either interacts directly with the methyl group (forming a methyl hydrogen bond)
                or replaces a structured water molecule surrounding the methyl group.</p>
            <br>
            <p><a class="link" target="_blank" href="http://nar.oxfordjournals.org/content/39/15/6741.long">doi:
                    10.1093/nar/gkr262</a></p>
        </div>
        <div id="coiled-coil" class="content-container">
            <span class="right"><img src="../assets/images/p66aMBD2_knobsholes.png" alt="">
                <p class="image-2">Hydrophobic “knobs” fit into “holes” as the two helices wrap around
                    one another. Key
                    charge interactions stabilize the complex and promote the anti-parallel arrangement.</p>
            </span>
            <span class="right">
                <script class="jmol" src="../structures/2l2l.js"></script>
                <p class="image-1">The <span class="cyan">MBD2</span> and <span class="purple">p66&alpha;</span>
                    proteins
                    from the NuRD complex form a canonical anti-parallel
                    heterodimeric coiled-coil complex.</p>
            </span>
            <h2>p66&alpha;:MBD2 coiled-coil complex</h2>
            <p>Based on previous work we postulated and showed that the coiled-coil domains of MBD2 and p66&alpha; form
                a
                stable heterodimeric complex. These domains bind one another with high affinity (~20 nM) yet largely
                remained as monomers in isolation. We determined the solution structure of p66&alpha;:MBD2 (<a
                    class="link" href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2L2L">2L2L</a>) which
                revealed a
                canonical anti-parallel coiled-coil complex with the typical “knobs into holes” hydrophobic interface
                stabilized by specific ionic interacions.</p>
            <br>
            <p>Based on this structure, we then showed that the p66&alpha; coiled-coil peptide could block the formation
                of
                the native NuRD complex and inhibit methylation-dependent gene silencing. This observation raises the
                possibility that a cell penetrating peptide or peptide mimetic could be developed for the treatment of
                cancer and β-hemoglobinopathies.</p>
            <br>
            <p><a class="link" target="_blank"
                    href="http://www.pnas.org/content/108/18/7487.long">doi:10.1073/pnas.1015341108</a>,<br><a
                    class="link" target="_blank"
                    href="http://www.jbc.org/content/288/5/3419.long">doi:10.1074/jbc.M112.431346</a>
            </p>
        </div>
        <div id="MBD3" class="content-container">
            <span class="right"><img src="../assets/images/MBD2vsMBD3.png" alt="">
                <p class="image-2">An overlay of <span class="purple">MBD3<sub>MBD</sub></span> with <span
                        class="green">MBD2<sub>MBD</sub></span> shows the structural similarity. A key difference
                    between the two domains involves a tyrosine to phenylalanine change in MBD3 which reduces
                    methylation specificity.</p>
            </span>
            <span class="right">
                <script class="jmol" src="../structures/2mb7.js"></script>
                <p class="image-1">The MBD3<sub>MBD</sub> adopts a the typical MBD fold with the central two strands of
                    a β-sheet forming a finger-like projection to extend down the major groove. NMR data, however,
                    indicated that the domain exchanged rapidly between CpG specific and non-specific binding modes.</p>
            </span>
            <h2>MBD3:hmCpG</h2>
            <p>We solved the sructure of MBD3<sub>MBD</sub> bound to DNA with a central hydroxymethylated CpG
                dinucleotide (<a class="link" target="_blank"
                    href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2MB7">2MB7</a>). The
                domain architecture is very similar to other MBDs (for comparison <b><span
                        class="purple">MBD3<sub>MBD</sub></span></b> is aligned to <b><span
                        class="green">MBD2<sub>MBD</sub></span></b> on DNA). The critical sequence specific
                tyrosine has been changed to a phenylalinine in MBD3 leading to a lack of methylation selectivity. As
                might be expected from the relative lack of sequence specificity, we found that MBD3<sub>MBD</sub> did
                not localize to a single binding site on the DNA but distributed between (m)CpG specific and
                non-specific binding modes.</p>
            <br>
            <p>We used chemical shift and residual dipolar coupling analyses to characterize the distribution of
                MBD3<sub>MBD</sub> on DNA. We found that MBD3<sub>MBD</sub> preferentially localizes to methylated CpG
                sites as compared to unmethylated and hydroxymethylated CpGs. Chemical shift changes also indicated that
                MBD3 slightly favored CpG and hmCpG sites as compared to non-specific DNA interaction. Localization
                depends on the density of (m)CpG sites such that the fraction bound to a mCpG goes down as the length of
                the DNA is increased.</p>
            <br>
            <span class="left"><img src="../assets/images/MBD3dist.png" alt="">
                <p class="image-3">MBD3<sub>MBD</sub> exchanges between <span class="red"> methylated </span> and <span
                        class="purple">unmethylated
                    </span> sites along the DNA, spending more time on the methylated site.</p>
            </span>
            <p>These findings lead to a model in which MBD3<sub>MBD</sub> distributes unevenly along the DNA, favoring
                sequences that contain multiple methylated and unmethylated CpG sites. In contrast, MBD2<sub>MBD</sub>
                exclusively localizes to methylated CpGs when DNA is methylated and more strongly localizes to
                unmethylated CpGs as compared to MBD3<sub>MBD</sub>. This model of MBD2/3 distribution on DNA correlates
                with recent studies showing that both MBD2 and MBD3 localize to unmethylated CpG islands while MBD2 more
                exclusively localizes to methylated CpG islands.</p>
            <br>
            <p><a class="link" target="_blank"
                    href="http://www.jbc.org/content/289/3/1294">doi:10.1074/jbc.M113.512236</a></p>
        </div>
        <div id="MBD4" class="content-container">
            <span class="right"><img src="../assets/images/MBD4Y109.png" alt="">
                <p class="image-2">A key tyrosine residue adopts a unique conformation in MBD4<sub>MBD</sub> such that
                    the sidechain points towards the phosphate backbone as opposed to interacting with the
                    methyl-cytosine as in <span class="pink">MeCP2<sub>MBD</sub></span> and other MBDs.</p>
            </span>
            <span class="right">
                <script class="jmol" src="../structures/2moe.js"></script>
                <p class="image-1">The MBD4<sub>MBD</sub> adopts a canonical MBD fold and binds to DNA with a similar
                    orientation as MBD2<sub>MBD</sub>.</p>
            </span>
            <h2>MBD4:mCpG</h2>
            <p>We determined the solution structure of MBD4<sub>MBD</sub> bound to methylated DNA (<a class="link"
                    target="_blank" href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2MOE">2MOE</a>).
                Similar to
                MeCP2<sub>MBD</sub>. the MBD4<sub>MBD</sub> contains a small insertion that leads to a longer C-terminal
                &alpha;-helix and larger hydrophobic core. Unlike other MBDs, however, the key tyrosine residue in
                MBD4<sub>MBD</sub> does not interact with the mC but is rotated towards the phosphate backbone. In the
                figures to the right, the orientation of Tyr<sup>109</sup> of MBD4 is compared to the equivalent
                Tyr<sup>123</sup> of MeCP2 which does interact with the mC. This relatively subtle structural change in
                MBD4<sub>MBD</sub> leads to an overall reduced binding affinity and selectivity for methylated DNA.</p>
            <br>
            <p>Unlike other members of the MBD family, MBD4 contains enzymatic activity within a C-terminal glycosylase
                domain (MBD4<sub>GD</sub>). The MBD4<sub>GD</sub> recognizes TpG·mCpG mismatches that arise from
                spontaneous deamination of mC. The N-terminal MBD4<sub>MBD</sub> is separated from the MBD4<sub>GD</sub>
                by approximately 300 amino acids which suggest that the MBD4<sub>MBD</sub> recruits the protein to
                methylated DNA and allows the MBD<sub>GD</sub> to identify and repair nearby mismatches. We have
                recently characterized inter- and intra-molecular exchange of MBD4<sub>MBD</sub> and found that this
                domain more efficiently exchanges between mCpG sites along the same DNA molecule as compared to separate
                DNA molecules. These observations are consistent with our model of MBD4<sub>MBD</sub> in which the
                primary function of this domain is to localize the protein to regions enriched for mCpG and then to
                rapidly exchange among binding sites.</p>
        </div>
    </div>
    <footer></footer>
</body>

</html>